modules/promotion/src/Entity/Promotion.php

```diff
@@ -240,6 +255,21 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayConfigurable('view', TRUE)
       ->setDisplayConfigurable('form', TRUE);
 
+    $fields['description'] = BaseFieldDefinition::create('string_long')
+      ->setLabel(t('Description'))
+      ->setDescription(t('The description of the promotion. This is additional information, beyond the name, which can be displayed to the customer.'))
```

Comment:
So Woo, Spree, etc don't have a helper label. "Additional information about the promotion to show to the customer"